examples/profile/profile.spec.js

```diff
@@ -41,6 +43,16 @@ describe('Sandbox Example', () => {
       .withAgeOver(18)
       .build();
 
+    const extraData = new SandboxExtraDataBuilder()
+      .withDataEntry(
+        new SandboxAttributeIssuanceDetailsBuilder()
+          .withIssuanceToken('some-token')
+          .withDefinition('some-definition')
+          .withExpiryDate(YotiDate.fromDateString('2020-01-01T00:00:00Z'))
```

Comment:
Same for this one, doesn't matter since it's Sandbox, but might be clearer to put a date in the future

Reply:
Fixed in 1bd4ad0